### PR TITLE
ticdc: add HTTP API for adjusting log level

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

add HTTP API for adjusting log level

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @TomShawn @leoppro

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -580,6 +580,16 @@ curl -X POST http://127.0.0.1:8300/capture/owner/move_table -d 'cf-id=cf060953-0
 }
 ```
 
+### 动态调整 cdc server 日志级别
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:583]: cdc 还是 TiCDC 呢？

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -580,6 +580,16 @@ curl -X POST http://127.0.0.1:8300/capture/owner/move_table -d 'cf-id=cf060953-0
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d '"debug"' http://127.0.0.1:8301/admin/log
+```
+
+POST 参数表示新的日志级别，支持 [zap 提供的日志级别](https://godoc.org/go.uber.org/zap#UnmarshalText)："debug", "info", "warn", "error", "dpanic", "panic", "fatal"。该接口参数为 json 编码，需要注意引号的使用：`'"debug"'`。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:591]:
```suggestion
`POST` 参数表示新的日志级别，支持 [zap 提供的日志级别](https://godoc.org/go.uber.org/zap#UnmarshalText)："debug"、"info"、"warn"、"error"、"dpanic"、"panic"、"fatal"。该接口参数为 JSON 编码，需要注意引号的使用：`'"debug"'`。
```

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (amyangfei) [inline ticdc/manage-ticdc.md:583]:
```suggestion
### 动态调整 TiCDC server 日志级别
```

Comment (amyangfei) [reply ticdc/manage-ticdc.md]: 更新了，使用 TiCDC server

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by leoppro — APPROVED** [review]
LGTM

**Pull request merged**